src/Microsoft.Health.Dicom.Core/Features/Workitem/Model/WorkitemInstanceIdentifier.cs

```diff
@@ -15,13 +15,15 @@ public class WorkitemInstanceIdentifier
         public WorkitemInstanceIdentifier(
             string workitemUid,
             long workitemKey,
-            int partitionKey = default)
+            int partitionKey = default,
+            long watermark = default)
```

Comment:
I don't see any changes to blob client file. Don't we need to update to take watermark in the filename?

Reply:
Done